### PR TITLE
New Enrichment Pages

### PR DESCRIPTION
**Pull request by msquance-stem**
## Status

* Current Status: Ready for review
* Closes https://github.com/NCCE/teachcomputing.org-issues/issues/2741

## Review progress:

- [ ] Browser tested
- [ ] Front-end review completed
- [ ] Tech review completed

## What's changed?

- New enrichment page all managed via strapi
- Includes new text with asides component from strapi dynamic zones

## Steps to perform after deploying to production

*If the production environment requires any extra work after this PR has been deployed detail it here. This could be running a Rake task, migrating a DB table, or upgrading a Gem. That kind of thing.*

### CODE REVIEW

File: app/components/cms_text_with_asides_component/cms_text_with_asides_component.html.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <%= render CmsRichTextBlockComponent.new(blocks: @blocks, with_wrapper: false) %>
+  <% end %>
+
+  <%= row.with_column("one-third") do %>
+
+    <% @aside_models.each do |aside| %>
+      <% if aside[:model] %>
+        <%= render aside[:model].render %>
+      <% else %>
+        <!-- missing aside <%= aside[:slug] %> -->
```
Comment (A-Wheeto) [inline app/components/cms_text_with_asides_component/cms_text_with_asides_component.html.erb:12]: Does this comment need removing?

**Review by A-Wheeto — CHANGES_REQUESTED** [review]
Looks great Mike, I noticed a comment that I assume needs to be removed?

File: app/components/cms_text_with_asides_component/cms_text_with_asides_component.html.erb (hunk 1 above)
Comment (msquance-stem) [reply app/components/cms_text_with_asides_component/cms_text_with_asides_component.html.erb:12]: Good question, I had it on the references to the hard coded asides we had in the dashboard, but I guess in the strapi version they should never be missing. I will remove it.

**Review by A-Wheeto — APPROVED** [review]
LGTM

**Pull request merged**